packages/vscode/src/hashes.ts

```diff
@@ -0,0 +1,36 @@
+import crypto from 'crypto'
+import { window, workspace } from 'vscode'
+
+/**
+ * Get a unique identifier for the project by hashing
+ * the directory with `schema.prisma`
+ */
+export async function getProjectHash(): Promise<string> {
+
+    let projectPath = await getSchemaPath()
+    projectPath = projectPath || process.cwd() // Default to cwd if the schema couldn't be found
+    console.log('projectPath: ' + projectPath)
+
+    return crypto
+        .createHash('sha256')
+        .update(projectPath)
+        .digest('hex')
+        .substring(0, 8)
+}
```

Comment:
This can be a feature in checkpoint itself, allowing to pass the project path dynamically. Probably an issue, for now, will suffice.

Reply:
Created an issue here: https://github.com/prisma/checkpoint-client/issues/45